README.md

```diff
@@ -1 +1,45 @@
 # sef-webapps
+
+# Setup Development Environment
+
+## Setting up Dependencies
+1. Setup WSO2 API Manager 2.6.0  (not 3.0.0)
+    https://docs.wso2.com/display/AM260/Installing+via+the+Installer
+    Download from this link : https://wso2.com/api-management/previous-releases/ (Version: 2.6.0) 
+2. Setup tomcat9 https://tomcat.apache.org/download-90.cgi
+3. Clone the sef common webapps repo https://github.com/sef-global/sef-webapps
+4. Create a new application in WSO2 APIM store and add the creadentials to the invoker.
+    
+    Go to WSO2 APIM Store: https://localhost:9443/store
+    
+    Go to Applications > Add new application > Pick a name > Click Add
+    
+    Go to the created application and click Generate button.
+    
+    Copy the key and secret to `sefwebapps/invoker/src/main/resources/configprops.properties` file
+    
+5. Build the invoker module using maven.
+    ```$xslt
+    mvn clean install
+    ``` 
+    Deploy the generated war in tomcat9 server.
```

Comment:
Move this step to under a new title. "Building the invoker"

Reply:
Done